sectorlisp.S

```diff
@@ -163,6 +175,27 @@ Cons:	xchg	%di,%cx				# Cons(m:di,a:ax):ax
 .RetDi:	xchg	%di,%ax
 	ret
 
+Builtin:cmp	$kAtom,%ax			# atom: last builtin atom
+	ja	.resolv				# ah is zero if not above
+	mov	(%si),%di			# di = Car(x)
+	je	.ifAtom
+	cmp	$kPrint,%al
+	je	.ifPrint
+	cmp	$kRead,%al
+	je	.ifRead
+	cmp	$kCons,%al
```

Comment:
can we reorder these?
.ifCar:	cmp	$kCar,%al
	je	Car
	cmp	$kCons,%al
	jb	Cdr
.ifCons:...

Reply:
I tried applying your change and it works for all of the interactive examples in my [sectorlisp-examples](https://github.com/woodrush/sectorlisp-examples) repo! Thanks! It shaved two more bytes. Could you send a pull request directly to this one, or after this one?

Reply:
I will send a PR after this is merged.